src/Popover/Popover.js

```diff
@@ -1,20 +1,18 @@
-/** @jsx jsx */
-import { jsx } from '@emotion/react';
-import React from 'react';
+import { createRef, cloneElement, Component } from 'react';
 import ReactPopover from 'react-popover';
 import PropTypes from 'prop-types';
 
 import PopoverContent from './PopoverContent';
 
-class Popover extends React.Component {
-  state = {
-    isOpen: false
-  };
-
+class Popover extends Component {
   constructor(props) {
     super(props);
 
-    this.node = React.createRef();
+    this.state = {
+      isOpen: false
+    };
```

Comment:
This and the movement of `toggle` below are just lints that popped up after removing the JSX pragma (`@jsx jsx`).